### ISSUE TITLE
hie-8.8.1 from unstable errors out at launch 

### ISSUE THREAD

**Issue by malob** [Infinisil/all-hies#43]
I tried installing `hie-8.8.1` which was recently added to `unstable`. It builds successfully on macOS 10.15.2, but produces the follow error when run.

```
dyld: Library not loaded: @rpath/libHShaskell-ide-engine-1.0.0.0-5EcNvjBz4sl3yA1Hn7NQQN-ghc8.8.1.dylib
  Referenced from: /Users/malo/.nix-profile/bin/hie-8.8.1
  Reason: no suitable image found.  Did find:
        /nix/store/gvdg0v5c0n8h0yi75w6k8gzn17wdr4q0-haskell-ide-engine-ghc881-970637fa13/lib/ghc-8.8.1/x86_64-osx-ghc-8.8.1/libHShaskell-ide-engine-1.0.0.0-5EcNvjBz4sl3yA1Hn7NQQN-ghc8.8.1.dylib: malformed mach-o: load commands size (33984) > 32768
```

**Comment by infinisil** [comment]
Seems like that's the issue in https://github.com/NixOS/nixpkgs/issues/22810, though https://github.com/NixOS/nixpkgs/pull/25537 is supposed to fix this. I'll compile it myself to check it out

**Comment by infinisil** [comment]
Apparently it's a problem in nixpkgs itself, see above PR for a fix. I'll update the nixpkgs used for GHC 8.8.1 once that change is in nixpkgs (and built by hydra)

**Comment by infinisil** [comment]
Should be fixed with 92148680060ed68f24738128d8489f4f9387d2ff :)

**Comment by malob** [comment]
Thanks! Can confirm it's working for me now 😄.
